Figaro/src/main/scala/com/cra/figaro/experimental/structured/ComponentCollection.scala

```diff
@@ -30,6 +30,12 @@ import com.cra.figaro.algorithm.factored.factors.Variable
  * To create a new component for an element, you need to say what problem it belongs to.
  */
 class ComponentCollection {
+  /** Indicates whether the old method of creating chain factors should be used.
+    * This defaults to false since the new method is generally better, but it's
+    * useful to have it available for comparison.
+    */
+  var useOldChainMethod = false
```

Comment:
Would it be possible to create a more descriptive name than 'new' and 'old' chain? These terms have meaning now, in December of 2016, but down the road it will become more difficult to understand them, especially to someone new to the codebase with no memory of how things worked before.